examples/hello_world/module/cpp/hello_world.cc

```diff
@@ -70,8 +70,16 @@ WASM_EXPORT void oak_main(uint64_t grpc_in_handle) {
       return;
     }
 
-    uint64_t rsp_handle;
-    channel_read(grpc_in_handle, buf, sizeof(buf), &actual_size, &rsp_handle, 1, &handle_count);
+    uint64_t handles[2];
+    channel_read(grpc_in_handle, buf, sizeof(buf), &actual_size, handles, 2, &handle_count);
```

Comment:
(pre_existing) rename `actual_size` to `buf_actual_size` and `handle_count` to `handle_actual_count`?

Reply:
Skipping this (I find the names clear enough already).